packages/server/graphql/mutations/endNewMeeting.ts

```diff
@@ -386,7 +383,8 @@ export default {
       teamId,
       isKill: getIsKill(meetingType, phase),
       updatedTaskIds,
-      removedTaskIds
+      removedTaskIds,
+      timelineEventId
```

Comment:
Why do you need to pass id this along?

Reply:
By passing `timelineEventId` here, `EndNewMeetingPayload` can use it with the dataLoader to grab the `timelineEvent`. If we don't pass the ID along, we won't have access to the `timelineEvent`.